modules/distr-tracing-otel-config-instrumentation.adoc

```diff
@@ -0,0 +1,239 @@
+////
+This module included in the following assemblies:
+-distr_tracing_otel/distr-tracing-otel-instrumentation.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-instrumentation_{context}"]
+= OpenTelemetry Instrumentation configuration options
+
+The {OTELName} can inject and configure OpenTelemetry auto-instrumentation libraries into your workloads. Currently, the project supports the injection of instrumentation libraries fro Go, Java, NodeJS, Python and ApacheHTTPD.
+
+Auto-instrumentation in OpenTelemetry refers to the capability where the framework automatically instruments an application, without
+necessitating manual code changes. This enables developers and operators to get observability into their applications with minimal effort and changes to the existing codebase.
+
+== Important
+
+Note that the {OTELName} operator product only supports the injection mechanism of the instrumentation libraries: not the instrumentation libraries or the upstream images. Customers can build their own instrumentation images or use the ones provided by the community.
+
+== Instrumentation options
+
+_sample OpenTelemetry collector custom resource file_
+
+[source,java]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: java
```

Comment:
a different name could be more suitable

Reply:
What do you think about `java-instrumentation`?